bundles/org.smarthomej.transform.basicprofiles/src/main/resources/OH-INF/i18n/basicprofiles.properties

```diff
@@ -0,0 +1,43 @@
+profile-type.basic-profiles.battery-low.label = Battery Low
```

Comment:
Please check these files. It looks like the languages are mixed (e.g. "Bildet bei Änderung... " in this file or  "Scale (non-negative number) to be applied on the state." in the german file).